pkg/logger/event/event.go

```diff
@@ -51,6 +52,21 @@ var ExtractNull ExtractAttribute = func(_ *restful.Request) (userID string, clie
 	return "", "", ""
 }
 
+func init() {
+	logrus.SetFormatter(UTCFormatter{&logrus.TextFormatter{TimestampFormat: millisecondTimeFormat}})
+}
+
+// UTCFormatter implements logrus Formatter for custom log format
+type UTCFormatter struct {
+	logrus.Formatter
+}
+
+// Format implements logrus Format for forcing time to UTC
+func (formatter UTCFormatter) Format(log *logrus.Entry) ([]byte, error) {
```

Comment:
Do we have a test for this?

Reply:
Ah forget to write test for this. I'll add test for this